README.md

```diff
@@ -315,3 +316,59 @@ Comp 40 gives you the tools to explore a large number of other topics:
+* Utilize the [Object Oriented Programming (OOP) concept](http://www.studytonight.com/cpp/cpp-and-oops-concepts.php) and learn how to use [pointers](http://cslibrary.stanford.edu/102/PointersAndMemory.pdf). Your code will become a lot more readable once you utilize these concepts and writing functions that are within 30 lines will become a lot easier. If the big ideas of COMP 15 are not clear, go to the office hours and get those covered ASAP! The way your code is graded will no longer be solely based on whether your code works. Your design matters too, as that is how it is done in industry, so really take the initiative to understand the big topics thoroughly!
+* Become comfortable with walking through how data is stored, accessed and manipulated each time you cover a new data structure. Draw them out with the TAs and see why each data structure has the time and space complexity that it has. In particular, seek the TAs to learn and visualize how the complex data structures work, instead of seeking them solely for debugging! The TAs for COMP 15 are an awesome group and all have a very deep understanding of how the data structures work. You will see that even one wrong detail in data structures will lead to segmentation faults or unexpected behavior. Each line of code you write really begins to matter, so even if you understand, go check with the TAs to make sure you are crystal clear!
+* Decisions, decisions, decisions! There are multiple right answers in data structures, which is why the question, "am I doing it right?" towards the TAs or the professors will give you a very mixed answer. Taking a well-informed decision after evaluating different options is a hallmark of a good engineer, so make sure you have a deep understanding of each data structure covered in class. There is nothing minor that you can skip in this class, as each new data structure is now a new option that you can consider for your upcoming homework or project. Each data structure has its own pros and cons, so learn how to evaluate and choose what you believe is the best for the problem you are trying to solve! 
+* There is no more HalliganHelper, but Piazza is always on to communicate with the TAs. Identify the location of the TAs through Piazza to get help! 
```

Comment:
Again, this is dependent on the professor. Current style is "no HalliganHelper" but that could change. I'd just remove this bullet point.